test/startsWith.test.js

```diff
@@ -0,0 +1,18 @@
+import test from 'ava'
+import {startsWith} from '../src'
+
+test('check correct input', t => {
+  const string = 'First PR'
+  const head = 'Fir'
+  const actual = startsWith(string, head)
+  const expected = true;
+  t.deepEqual(actual, expected)
+})
+
+test('check wrong input', t => {
+  const string = 'First PR'
+  const head = 'Foo'
+  const actual = startsWith(string, head)
+  const expected = false;
+  t.deepEqual(actual, expected)
```

Comment:
Yep, I guess it's kinda silly to `deepEqual` bolleans... 😃 . I need to dive into *ava* API and syntax.
I'll update the code ASAP. 

Thanks a lot Kent!